### PR TITLE
New format for most of the code

### PR DESCRIPTION
**Pull request by Null-B**
update for the code of the botcode
and a good amout of new html to the readme 
filip needs to update his change log when he dose something

the 2 main thing that are new are for the botcode and readme

### CODE REVIEW

**Comment by Null-B** [comment]
filip do more codeign you lazy fuck

**Comment by filipgaming008** [comment]
The README.md and .gitignore are good. but i do not like the rest of the fixes to the code, where you add spaces between "=" for variables.

File: botcode.py [hunk 1]
```diff
@@ -80,90 +79,98 @@ async def checkmodules(ctx, module_name):
...
+  # unload bot_cogs error handle
+
 @unload.error
 async def unload_error(ctx, error):
     if isinstance(error, commands.MissingPermissions):
         await ctx.send("You cant do that!")
 
-# load all Cogs on start command
+  # load all Cogs on start command
+
```
Comment (Null-B) [inline botcode.py]: no

**Review by filipgaming008 — APPROVED** [review]
approved by professional gopnik. pog edits. yues.

**Comment by Null-B** [comment]
> approved by professional gopnik. pog edits. yues.

no

**Pull request merged**